packages/arb-util/value/tuple.go

```diff
@@ -251,6 +253,7 @@ func (tv TupleValue) String() string {
 
 func (tv TupleValue) HashPreImage(firstHash common.Hash, size int64) common.Hash {
 	return hashing.SoliditySHA3(
+		hashing.Uint8(TypeCodeTuple),
 		hashing.Bytes32(firstHash),
 		hashing.Uint256(big.NewInt(size)),
 	)
```

Comment:
This function is weird since we doesn't actually get any data out of the tuple. Do we actually need it or could it just be deleted. Functionality seems to be the same as `HashPreImage.Hash()`